css/products.css

```diff
@@ -0,0 +1,18 @@
+
+.sidebar-container{
+    min-width: 15rem;
+    padding:1.5rem;
+}
+
+.cards-container{
+    flex-wrap: wrap;
```

Comment:
you can add this in your utilities as it will repeat multiple times

Reply:
Added the utility class flex-wrap